src/model/model.ts

```diff
@@ -182,7 +183,7 @@ export class Model {
       const ast = astMap.get(key);
       let rm = rmMap.get(key);
       if (!rm) {
-        rm = new DefaultRoleManager(10);
+        rm = this.synced ? new DefaultSyncedRoleManager(10) : new DefaultRoleManager(10);
```

Comment:
If rm is undefined or null, we should throw an error,  we ensure that rm has been initialized in enforcer.